Gemfile.lock

```diff
@@ -40,5 +40,8 @@ DEPENDENCIES
   rake
   rspec
 
+RUBY VERSION
+   ruby 2.3.1p112
+
 BUNDLED WITH
-   1.10.6
```

Comment:
As far as I know, the changes to the `Gemfile.lock` should be removed, with the reasoning being something to do with the version of bundler specified from puppet.